governance-exercises/CivicSage Exercises/Ex01/3.1 Adjust Spork D3M Parameters - JULY 24, 2023.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Adjust Spork D3M Parameters - JULY 24, 2023
+summary: Signal your support or opposition to adjusting the Spork DAI Direct Deposit Module (D3M) parameters.
+discussion_link: https://github.com/makerdao/community/blob/master/governance-exercises/poll-exercise-01.md
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-07-24T16:00:00
+end_date: 2023-07-27T16:00:00
+---
+# Poll: Adjust Spork D3M Parameters - JULY 24, 2023
+
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of Spork. The community can vote in this poll to express support or opposition to using the Spork Direct Deposit DAI Module (D3M) with the listed parameters. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, July 24 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to making the following parameter changes to the Spork DAI Direct Deposit Module (D3M):
+* Increase the DIRECT-Spork-DAI [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 50 million DAI to 100 million DAI.
+* Increase the DIRECT-Spork-DAI [Target Available Debt](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) from 5 million DAI to 15 million DAI.
+* Increase the DIRECT-Spork-DAI [Ceiling Increase Cooldown](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) from 28,000 seconds to 86,400 seconds.
+
+Please review the discussion [thread](ttps://github.com/makerdao/community/blob/master/governance-exercises/poll-exercise-01.md) to help inform your position before voting.
+
+For more information regarding the DAI Direct Deposit Module (D3M) and how these parameters interact, check out the forum thread [here](https://forum.makerdao.com/t/discussion-direct-deposit-dai-module-d3m/7357).
```

Comment:
The template link is outdated, we should use this link: https://manual.makerdao.com/module-index/module-dai-direct-deposit